storage/innobase/log/log0recv.cc

```diff
@@ -4142,6 +4142,32 @@ static
   return end_lsn;
 }
 
+#ifdef XTRABACKUP
+/** recalucate file start_lsn by scanning the first block
+and calculationg the start LSN
+@param[in,out]  log                     redo log
+@param[in,out]  checkpoint_lsn          log sequence number found in checkpoint
```

Comment:
only [out]

Reply:
We are not return LSN from checkpoint header. Please fix.
AFAIU, we are returning the LSN of the first block in the redo file.

Reply:
checkpoint_lsn is the wrong word use. first_lsn may be.

Reply:
fixed, added new variable start_lsn and calculated_start_lsn